.teamcity/settings.kts

```diff
@@ -20,12 +20,10 @@ object RhinoLicensingBuild : BuildType({
         param("teamcity.git.fetchAllHeads", "true")
         param("env.Git_Branch", "${RhinoLicensingVcs.paramRefs.buildVcsBranch}")
```

Comment:
```suggestion
        param("env.Git_Branch", "%teamcity.build.vcs.branch.RhinoLicensingAnsible_RhinoAnsibleVcsRoot%")
```

This line was erroring on the import now, and this is the format used by other builds using `env.Git_Branch`